source/mongodb-handshake/handshake.rst

```diff
@@ -355,8 +371,19 @@ be populated:
 Missing variables or variables with values not matching the expected type MUST cause the
 corresponding ``client.env`` field to be omitted and MUST NOT cause a user-visible error.
 
-The contents of ``client.env`` MUST be adjusted to keep the handshake below the size limit;
-see `Limitations`_ for specifics.
+Container
+^^^^^^^^^
+
+Container runtime information is captured in ``client.env.container``.
+
+``client.env.container.runtime`` SHOULD be set to ``"docker"`` if the file ``.dockerenv``
```

Comment:
What's the reason for using `SHOULD` here?  Usually we opt for `MUST` in specifications unless there are reasons to not mandate it. I don't see why these should be optional if we detect a container runtime or orchestrator.

```suggestion
``client.env.container.runtime`` MUST be set to ``"docker"`` if the file ``.dockerenv``
```